src/runner/browser-set.js

```diff
@@ -44,16 +45,24 @@ export default class BrowserSet extends EventEmitter {
         await promisifyEvent(bc, 'closed');
     }
 
-    _waitConnectionsOpened () {
+    async _getReadyTimeout () {
+        var isLocalBrowserPromises = this.connections.map(connection => connection.provider.plugin.isLocalBrowser());
```

Comment:
I don't know details of implementation, but it looks like you can subtract the number of ```localBrowserPromises``` from number of connections. Or even 
```javascript
var remoteBrowserExists = this.connections.map(connection => !connection.provider.plugin.isLocalBrowser());
```
Otherwise, variable name is confusing me.